dashboard/src/test/java/com/google/cloud/tools/opensource/dashboard/DashboardTest.java

```diff
@@ -212,11 +213,14 @@ public void testDashboard_recommendedCoordinates() {
     Nodes recommendedListItem = dashboard.query("//ul[@id='recommended']/li");
     Assert.assertTrue(recommendedListItem.size() > 100);
 
-    List<String> coordinateList =
-        toList(recommendedListItem).stream().map(Node::getValue).collect(toImmutableList());
     // fails if these are not valid Maven coordinates
-    coordinateList.forEach(DefaultArtifact::new);
+    for (Node node : recommendedListItem) {
+      new DefaultArtifact(node.getValue());
+    }
 
+    ImmutableList<String> coordinateList =
+        Lists.newArrayList(recommendedListItem).stream().map(Node::getValue).collect(toImmutableList());
```

Comment:
```suggestion
        Streams.stream(recommendedListItem).map(Node::getValue).collect(toImmutableList());
```

From `com.google.common.collect.Streams`.

Reply:
done